src/utils.js

```diff
@@ -235,3 +235,30 @@ export function apiToMessage(string) {
 export function isBrowserNamespace(string) {
   return ['browser', 'chrome'].includes(string);
 }
+
+
+export function parseCspPolicy(policy) {
```

Comment:
There's https://www.npmjs.com/package/csp-parse and a few others which are all essentially the same or similar to this. I didn't want to pull in an external library for such an easy part.